Jenkinsfile

```diff
@@ -81,14 +81,5 @@ pipeline {
                 }
             }
         }
-        
-
-         stage('Deploy client & server (master only)') {
-            when { branch 'master'}
-            steps {
-                 build job: 'deploy-npm-glsp-examples', wait: false
```

Comment:
I am not clear on what the reasoning is to remove the build jobs do we not need them anymore? Could you please provide some details on that?

Reply:
As discussed here, we currently are not planning on providing nightly deployment for the example repository.  Only releases are going to be deployed to npm/maven and this has to be done manually anyways. (At least for now) So we don need these build jobs anymore